src/biogeophys/LunaMod.F90

```diff
@@ -404,22 +404,20 @@ subroutine Update_Photosynthesis_Capacity(bounds, fn, filterp, &
                          PNcbold   = 0.0_r8                                     
                          call NitrogenAllocation(FNCa,forc_pbot10(p), relh10, CO2a10, O2a10, PARi10, PARimx10, rb10v, hourpd, &
                               tair10, tleafd10, tleafn10, &
-                              Jmaxb0, Jmaxb1, Wc2Wjb0, relhExp, PNlcold, PNetold, PNrespold, dayl_factor(p),&
-                              PNcbold, PNstoreopt, PNlcopt, PNetopt, PNrespopt, PNcbopt)
+                              Jmaxb0, Jmaxb1, Wc2Wjb0, relhExp, PNlcold, PNetold, PNrespold, PNcbold, &
+                              dayl_factor(p), PNstoreopt, PNlcopt, PNetopt, PNrespopt, PNcbopt)
                          vcmx25_opt= PNcbopt * FNCa * Fc25
                          jmx25_opt= PNetopt * FNCa * Fj25
                           
                          chg = vcmx25_opt-vcmx25_z(p, z)
                          chg_constrn = min(abs(chg),vcmx25_z(p, z)*max_daily_pchg)
-                         vcmx_prevyr(p,z) = vcmx25_z(p,z)
                          vcmx25_z(p, z)  = vcmx25_z(p, z)+sign(1.0_r8,chg)*chg_constrn
-                         vcmx_prevyr(p,z) = (vcmx_prevyr(p,z)+vcmx25_z(p,z))/2.0_r8
+                         vcmx_prevyr(p,z) = vcmx25_z(p,z)
```

Comment:
@lmbirch89 & @billsacks I think we're on the same page, that we should be using that last vcmx values from the previous year.  Is the implementation here appropriate?  Is the variable name OK?  Ditto for line 420.

Reply:
@ekluzek I think @dlawrenncar and @wwieder would like to bring this on a the LUNA bug fix branch #961 so that @olyson can help us evaluate the effect of the bug.  Do I need to create a new issue to help track this?

Reply:
I think the variable name is maybe the last thing to get right here. "prevyr" is probably OK. But, maybe something like "lastvalid"? or "endoflastyr"?

Reply:
And @wwieder if we are going to call this a "bug" we should create an issue for it. I can use the code here to figure out what to put on the bug fix branch. I suppose I can use the current implementation that uses the "_prevyr" suffix, until we decide on a better name.

Reply:
I think we want to evaluate the implementation before bringing this change in as a bug fix.  I'll create an issue

Reply:
I'm happy with the implementation here, but:

1. I find that having "yr" / "year" in the name here is misleading: We would normally use "year" in the name of a variable for which something is actually happening on an annual time step, or with an annual time scale. Here, at any given time, this variable gives the value from the last day, not from the last year. This could be confusing for someone trying to understand the code or reuse this variable for a different purpose. (I had one possible name in https://github.com/ESCOMP/CTSM/pull/947#discussion_r406975475 but I don't know if that's the best.)

2. I'd like confirmation from someone that the initial values of these variables (30 and 60) is what's desired: see my last paragraph in https://github.com/ESCOMP/CTSM/pull/947#discussion_r406975475

Reply:
@billsacks I had forgotten to talk about the initialization of jmax_prevyr and vcmax_prevyr. I chose 60 and 30 because they represented Arctic PFTs better, according to the PFT specific output. To be backwards compatible, there is probably an argument for choosing the original spring values of jmax and vcmax (80 and 55). However, for a global initialization, I'm not sure how correct it is or how much bias it would introduce in a simulation less than 1 year.

Reply:
Let's make sure we get the right initialization values for these in this branch.